lib/lotus/commands/generate/migration.rb

```diff
@@ -0,0 +1,32 @@
+require 'lotus/utils/string'
+require 'lotus/generators/migration'
+
+module Lotus
+  module Commands
+    module Generate
+      class Migration 
+        
+        attr_reader :app_name, :migration_name, :migration_time, :migration_class, :cli, :source, :target
+
+        def initialize(name, environment, cli)
+          name = Lotus::Utils::String.new(name)
+          #@app_name = ""
+          @migration_name = name.underscore
+          @migration_time = Time.now.to_i.to_s
```

Comment:
Timestamp should be human readable, what do you think about formatting it as `ddmmyyyy`?